about.html

```diff
@@ -0,0 +1,113 @@
+                    page when looking at its layout. The point of using Lorem 
+                    Ipsum is that it has a more-or-less normal distribution of 
+                    letters, as opposed to using 'Content here, content here',
+                     making it look like readable English.
+                     </p>
+                <p class="about-contact-text">Please Contact us per Email for any further questions about Afghanistan Developer seminar</p>
+                <a class="email-link" href=""><p>kazim.mohammadi110@gmail.com</p></a>
+            </div>
+        </section>
+        <section class="about-logo-section">
+            <div>
+                <h3>Afghanistan Developers Confrenece</h3>
+                <p class="about-logo-disc">The logo of Afghanistan Developers was decided
+                 through the logo competation from 8. Juneto 7. July
+                </p>
+                <p class="about-logo-cont">
+                    The point of using Lorem Ipsum 
+                    is that it has a more-or-less normal distribution of 
+                    letters, as opposed to using 'Content here,
+                     content here', making it look like readable English.
+                </p>
+            </div>
+            <img class="about-logo-img" src="./img/afghan-logo.png" alt="">
+        </section>
+        <section class="history-section">
+            <div>
+                <h3>See the past Afghan Dev Confrenece</h3>
+                <p>Take a look at the last two Afghanistan Developers 
+                    Confrenece whick took place in Busines 
+                    Aines and in Warsaw
+                </p>
+                <div class="history-img-cards">
+                    <div class="history-card card1">
+                        <div class="history-layer">
+                            <h4 class="history-date">2018</h4>
+                            <p>Afghanistan Developers Confrenece in Herat</p>
+                        </div>
+                    </div>
+                    <div class="history-card card2">
+                        <div class="history-layer">
+                            <h4 class="history-date">2020</h4>
+                            <p>Afghanistan Developers Confrenece in Mazar</p>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </section>
+    </main>
+    <footer>
+        <div class="footer-main-cont">
+            <div  class="top-footer">
+                <h3 class="footer-title">Partner</h3>
+              <div class="footer-logos-container">
+                <img src="./img/hbo.png" alt="">
+                <img src="./img/cloudsmith.png" alt="">
+                <img src="./img/nanohub.png" alt="">
+                <img src="./img/motocycles.png" alt="">
+                <img src="./img/walmart.png" alt="">
+              </div>
+            </div>
```

Comment:
- [ ] The partner section is not shown in the desktop version of the about page, kindly consider making this section hidden for desktop view